lib/mllp/sender.ex

```diff
@@ -1,10 +1,8 @@
 defmodule MLLP.SenderContract do
-  @type sender_error ::
-          :unspecified_error
```

Comment:
These atoms were not used,  so I removed them, but can put them back off if there's a reason they were there that I'm not seeing.